### PR TITLE
Minimized auth & DB + auto invite for Tychobra/POW emails (corresponds w/ powpolishedapi PR)

### PR DESCRIPTION
**Pull request by phoward38**
- PR for corresponding `powpolishedapi` [PR](https://github.com/powwater/powpolishedapi/pull/1)

### CODE REVIEW

**Comment by jimbrig** [comment]
@phoward38 I will get to this review this afternoon.

File: R/global_sessions_config.R [hunk 1]
```diff
@@ -52,8 +54,9 @@ global_sessions_config <- function(
   firebase_config = NULL,
   admin_mode = FALSE,
   is_invite_required = TRUE,
-  api_url = "https://api.powwater.org",
+  api_url = "https://api.powwater.com",
```
Comment (jimbrig) [inline R/global_sessions_config.R:57]: have you switched/forwarded the domain to `.com`? if so can you show me how to implement that for all deployments?

**Review by jimbrig — APPROVED** [review]
Good to go here

File: R/global_sessions_config.R (hunk 1 above)
Comment (phoward38) [reply R/global_sessions_config.R:57]: No, we'll need to use `.org` for now

**Pull request merged**